### PR TITLE
Bump Angular version 12.1.0

### PR DESCRIPTION
**Pull request by aprilmay**

### CODE REVIEW

**Comment by HarelM** [comment]
I think the commit in the upstream also has partial support for ivy which I think is important, doesn't it?

**Comment by HarelM** [comment]
Thanks for taking the time to make the necessary choose changes!
Do you understand why the build fails? Does it fail for you locally too?
CI is not using yarn, which might explain some issues I believe...?

**Comment by HarelM** [comment]
I can't find a change to the supported angular version in the project's package.json file:
https://github.com/maplibre/ngx-maplibre-gl/blob/f3278461ebac94c15318f9e74f0c8718a8c7c6fd/projects/ngx-maplibre-gl/package.json#L23

**Comment by aprilmay** [comment]
The build was failing because i failed to upgade `@angular-devkit/build-angular`. It was OK locally. Nothing to do with yarn. Ivy should be on :)

**Comment by HarelM** [comment]
Thanks!

File: projects/ngx-maplibre-gl/src/public_api.ts [hunk 1]
```diff
@@ -4,8 +4,26 @@
...
+export * from './lib/source/raster-source.component';
+export * from './lib/source/geojson/feature.component';
+export * from './lib/marker/marker.component';
+export * from './lib/popup/popup.component';
+export * from './lib/control/fullscreen-control.directive';
+export * from './lib/control/geolocate-control.directive';
+export * from './lib/control/navigation-control.directive';
+export * from './lib/control/attribution-control.directive';
+export * from './lib/control/scale-control.directive';
+export * from './lib/markers-for-clusters/markers-for-clusters.component';
```
Comment (HarelM) [inline projects/ngx-maplibre-gl/src/public_api.ts:26]: Are all these exports really required? Why do we need to export all of them?

Comment (aprilmay) [reply projects/ngx-maplibre-gl/src/public_api.ts:26]: I assume that the idea (from upstream) is to give the ability to allow custom controls and other components to extend (subclass) from these classes, and no that there's no harm anyway

Comment (HarelM) [reply projects/ngx-maplibre-gl/src/public_api.ts:26]: Upsteam has this for geocoder I think. But yea, it probably can't hurt...

**Pull request merged**